code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -1146,6 +1146,15 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_desc = "A drink from Valhalla."
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 
+/datum/reagent/consumable/ethanol/mead/on_mob_life(mob/living/carbon/M, delta_time, times_fired)
+	holder.add_reagent(/datum/reagent/consumable/sugar, 3 * REM * delta_time)
+	if(DT_PROB(33, delta_time))
+		M.adjustBruteLoss(-1, 0)
+		M.adjustFireLoss(-1, 0)
+		M.adjustOxyLoss(-1, 0)
+		M.adjustToxLoss(-1, 0)
+	return ..()
+
```

Comment:
This is not a request to change, I just want to discuss:

This is a duplicate of the behaviour of the honey reagent, obviously deliberately, because mead is made with honey.

But what if the behaviour of honey changes, and someone forgets to change this?

Do we split this simplified healing and sugar production off into its own proc? Avoids repeating, but seems excessive.

Having mead "decompose" to honey means that you get the nutritional value of honey from drinking mead, thus making mead just "honey but also gets you drunk".

Am I over thinking this?

Reply:
DRY

Reply:
Okay, here's the request:

Make a new reagent, let's call it "Metabolized Honey", which both mead and honey decompose into, which then has the "adds sugar and does generic healing" effects.